### ISSUE TITLE
[Feature] Live marker preview

### ISSUE THREAD

**Issue by 0x60B2** [stashapp/stash#5448]
**Is your feature request related to a problem? Please describe.**
Marker previews take up a lot of space. Say you have:

60k markers * 10 MiB (20 sec video) = 600 GiB

**Describe the solution you'd like**
A way to only generate marker image preview, and when hover, just stream the actual video.

**Describe alternatives you've considered**
Not generating marker previews

**Additional context**
This could also be used for scenes I guess.

**Comment by UncertainMongoose** [comment]
I second this. If it's possible, it would be very advantageous.

**Comment by Harbingerhq** [comment]
I agree, this would be really good

**Comment by trekGriffin** [comment]
yes. and if only generate the preview that I wanted tag's marker is the best.

**Comment by mf35** [comment]
Would this feature be conceivable given the recent introduction of timed markers ? #5311 
This opens the way to interesting features, such as creating playlists based on markers.
